### DISCUSSION TITLE
sonoff dongle E router

### DISCUSSION THREAD

**Discussion by Rodrigorjn** [Koenkk/zigbee2mqtt#19016] [Devices]
Hello, I would like to know if anyone has already managed to use the sonoff dolgle E as a repeater on zeegbee2mqtt?

**Comment by francisp2** [comment]
You have to flash it with router firmware, then it works fine.

**Reply by Rodrigorjn** [reply]
yes, but is there firmware for the sonoff E router?

**Reply by kirovilya** [reply]
for example https://github.com/xsp1989/zigbeeFirmware/tree/master/firmware/Zigbee3.0_Dongle-NoSigned/RouterForDongle
